chaincode/tuple_aggregate.go

```diff
@@ -318,6 +327,15 @@ func logFailAggregate(db *LedgerDB, args []string) (outputAggregatetuple outputA
 	if err != nil {
 		return
 	}
+
+	// cancel aggregatetuple if compute plan is canceled
+	if aggregatetuple.ComputePlanID != "" {
+		err := cancelIfComputePlanIsCanceled(db, inp.Key, aggregatetuple.ComputePlanID, &aggregatetuple)
+		if err != nil {
+			return outputAggregatetuple{}, err
```

Comment:
ok that's wayyy to many times I copy/pasted my comment already :D I'll stop now.

Reply:
It would work at every time you copy/pasted it indeed but I think it's intentional here. @inalgnu will have to confirm it but return without explicit arguments are considered confusing and to be avoid.
Named return values are for documentation mostly but even then you should specified the return arguments. Here it would be `return o, err` at the very least.
It's mostly my bad if you've find a lot of bare return around the base code. Sorry about that.

Reply:
Interesting, thanks for the precision! Let's see what @inalgnu says.

Reply:
@AurelienGasser named params could be easy to use but might cause some issues especially shadowing. I used to avoid using them as a good practice.
this won't compile for example => https://play.golang.org/p/2rumK4hdKec
from codereviewcomments:
https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

> Don't name result parameters just to avoid declaring a var inside the function; that trades off a minor implementation brevity at the cost of unnecessary API verbosity.

In addition, here outputAggregatetuple type was named same as the parameter name which leads to an error if you want to create a new struct outputAggregatetuple{}
https://play.golang.org/p/g0RuGAR6W9t

Reply:
Thank you, and sorry for the avalanche of comments!